vr-wordle/src/Components/Grid.js

```diff
@@ -45,8 +45,6 @@ function StaticLetter(props) {
       position={props.position}
       scale={1}
```

Comment:
do you need to specify scale={1} or is this default?

Reply:
I think you're right, that is the default, it should be removed
```suggestion
```